include/swift/AST/ModuleDependencies.h

```diff
@@ -992,7 +992,7 @@ class SwiftDependencyScanningService {
   std::shared_ptr<llvm::cas::ActionCache> ActionCache;
 
   /// File prefix mapper.
-  std::unique_ptr<llvm::PrefixMapper> Mapper;
+  std::shared_ptr<llvm::PrefixMapper> Mapper;
```

Comment:
Can this be moved to worker? I feel like this should be shared, probably only here due to some unfortunate layering too.

Reply:
Resolver holds an instance. We can totally rebuild the map from a CompilerInstance and we should probably do that to fully decouple the layering.

Reply:
We can maybe even use the CAS from the instance, if that is possible.